pallets/anmol-utils/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use sp_std::vec::Vec;
+
+pub fn remove_vector_item<'a, T: Ord>(vector: &'a mut Vec<T>, item: &T) -> Result<T, &'static str> {
+	match vector.binary_search(item) {
+		Ok(index) => Ok(vector.remove(index)),
+		Err(_) => Err("RemoveVectorItem"),
+	}
+}
```

Comment:
CC @justinFrevert

Reply:
Seems fine to me. It may be idiomatic to move it over and keep its crate structure https://stackoverflow.com/questions/44539729/what-is-an-idiomatic-way-to-have-shared-utility-functions-for-integration-tests

Reply:
Yes, we should keep it as a crate. Maybe, the path could be `/shared/util`. This will allow us to add other crates if we seem necessary.

Reply:
Just saw that you added to a subfolder in `/common`. Looks good like that as well.